src/SqlUtils.ts

```diff
@@ -55,4 +54,51 @@ export default class SqlUtils {
         return ipAddress;
     }
 
+    /**
+     * Opens a new connection to the database and executes a T-SQL script on it.
+     * @param connectionConfig Config object for the connection
+     * @param command T-SQL to be executed on the connection
+     */
+    static async executeSql(connectionConfig: SqlConnectionConfig, command: string): Promise<void> {
+        let pool: mssql.ConnectionPool | undefined;
+        try {
+            pool = await mssql.connect(connectionConfig.Config);
+            const result = await pool.query(command);
+            
+            // Display result
+            for (let i = 0; i < result.recordsets.length; i++) {
+                console.log(`Rows affected: ${result.rowsAffected[i]}`);
+                // Displays query result as JSON string. Future improvement: IRecordSet has function toTable() 
+                // which returns more metadata about the query results that can be used to build a cleaner output
+                console.log(`Result: ${JSON.stringify(result.recordsets[i])}`);
+            }
+        }
+        catch (connectionError) {
+            if (connectionError instanceof mssql.MSSQLError) {
+                this.reportMSSQLError(connectionError);
+                throw new Error('Failed to execute query.');
+            }
+            else {
+                // Unknown error
+                throw connectionError;
+            }
+        }
+        finally {
+            if (!!pool) pool.close();
```

Comment:
can this be simplified to `pool?.close()`? If not, please add `{}` after the if statement